wolfSSL/src/chapter02.md

```diff
@@ -536,10 +716,180 @@ Enables Online Certificate Status Protocol (OCSP) support.
 
 Builds even more OpenSSL compatibility into the library, and enables the wolfSSL OpenSSL compatibility layer to ease porting wolfSSL into existing applications which had been designed to work with OpenSSL. It is off by default.
 
+#### HAVE_EXT_CACHE
+
+OpenSSL compat layer. Need OPENSSL_EXTRA to be defined as well.
+
+#### WOLFSSL_WPAS_SMALL
+
+WPA supplicant support. Used with OPENSSL_EXTRA for compatibility use.
```

Comment:
This enables a smaller subset of the compatibility layer for WPA supplicant support.